examples/integrations/gcp/README.md

```diff
@@ -24,38 +24,42 @@ Run ETL jobs, train and deploy models, and track the performance using this code
 2. Run `./build.sh` to build the container
 2. Making a request to the service (find an example in `test/test_local.py`) in another shell
```

Comment:
I guess I didn't catch this before, but there is no test/test_local.py, is there?

Reply:
Good catch. Fixed